server/routes/mpapi.ts

```diff
@@ -0,0 +1,52 @@
+import type { RequestOptions } from 'http';
+import https from 'https';
+import type { NextFunction, Request, Response } from 'express';
+import { Router } from 'express';
+import { log } from '../log';
+import { withIdentity } from '../middleware/identityMiddleware';
+import { getConfig, getOptions } from '../mpapiConfig';
+import { handleError, mimicResponse } from '../util';
+
+const router = Router();
+
+router.use((_req: Request, _res: Response, next: NextFunction) => {
+	next();
+});
+
+const makeMpapiRequest = (options: RequestOptions, res: Response) => {
+	const mpapiRequest = https.request(options, (mpapiResponse) => {
+		mimicResponse(mpapiResponse, res);
+		mpapiResponse.pipe(res);
+	});
+
```

Comment:
Linter - spacing?

Reply:
I don't see the problem? es-lint and prettier have been run on it too